platforms-serverless/netlify-cli/test.sh

```diff
@@ -3,9 +3,9 @@
 set -eux
 
 if [ "$PRISMA_CLIENT_ENGINE_TYPE" == "binary" ]; then
-  BINARY_STRING=',"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
+  BINARY_STRING=',"files":["edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]'
```

Comment:
These are now always included in the Client?

(Answer as comment, not emoji reaction please so I get notification. Thanks.)

Reply:
Yes, that's the side-effect of the removal of `--data-proxy` and also because it is a custom output.

Reply:
Surprising that this did not affect more files/expectations.

Reply:
Non-custom output already had a setup edge client that just throws an error, but that did not exist for custom outputs.